blahaj/data.lua

```diff
@@ -1,57 +1,61 @@
+require("sounds.soundvariation")
+
 data.raw["capsule"]["raw-fish"].localised_name = "BLÅHAJ"
-data.raw["capsule"]["raw-fish"].localised_description = "A cute BLÅHAJ. Left click to cuddle with it and gain some health."
+data.raw["capsule"]["raw-fish"].localised_description =
+    "A cute BLÅHAJ. Left click to cuddle with it and gain some health."
 
-data.raw["capsule"]["raw-fish"].icon = "__blahaj__/graphics/blahaj/blahaj_full.png"
+data.raw["capsule"]["raw-fish"].icon =
+    "__blahaj__/graphics/blahaj/blahaj_full.png"
 data.raw["capsule"]["raw-fish"].icon_size = 512
 data.raw["capsule"]["raw-fish"].capsule_action = {
-  type = "use-on-self",
-  uses_stack = false,
-  attack_parameters = {
-    type = "projectile",
-    activation_type = "activate",
-    range = 0,
-    cooldown = 120,
-    ammo_type = {
-      category = "capsule",
-      target_type = "position",
-      action = {
-        type = "direct",
-        action_delivery = {
-          type = "instant",
-          target_effects = {
-            {
-              type = "damage",
-              damage = {
-                amount = -20,
-                type = "physical"
-              }
+    type = "use-on-self",
+    attack_parameters = {
+        type = "projectile",
+        activation_type = "consume",
```

Comment:
Hey don't change this. We don't want to eat the blahaj.

Reply:
Ohhh, I thought I had broken it, I didn't realise it wasnt meant to be consumable